test/runnable/mangle.d

```diff
@@ -589,6 +589,99 @@ void fooB(void delegate (void delegate()) scope dg)
 //pragma(msg, fooB.mangleof);
 static assert(typeof(fooA).mangleof != typeof(fooB).mangleof);
 
+/***************************************************/
+
+@("test overriding mangling for alias declaration")
+struct TestOverridingManglingAlias
+{
+    enum baseMangling = "6mangle27TestOverridingManglingAlias";
+
+    @("basic type")
+    {
+        @("override mangling")
+        {
```

Comment:
What is the purpose of all of these `@("some string")` UDAs?

Reply:
It adds a description to each test indicating what is tested. If you don't like it I can remove it. I think it adds value.

Reply:
I don't see where those descriptions are being used anywhere.  How is this different from just adding a comment?

Reply:
They could be used in the future and they allow me to add a form of scope for the description.

Reply:
I think until we have something in our infrastructure that makes use of these descriptions, they're a bit of a distraction.

Reply:
Fine 😞. I'll remove or move them to comments.

Reply:
Done.

Reply:
I think that now it's much more difficult to see which code belongs to which test and which description belongs to which test. In any other D project I would have used a separate unit test block for each test, but the DMD tests are not using unit test blocks.

Reply:
You can easily resolve that with some kind of comment header...

```D
/*******************************************************
*  Section 1
*******************************************************/

// Section 1.1

// Section 1.2

/*******************************************************
* Section 2
*******************************************************/

// Section 2.1

// Section 2.2
```
... or something along those lines

FWIW, I see the value in what you were proposing, but I don't think this PR is the right place to introduce such idioms into the code base unless it solves a specific problem that we're currently facing and/or has broader support/adoption.

Reply:
I think it solves specific problem we have but nobody else seems to agree with me.